Homework/2/Program.cs

```diff
@@ -0,0 +1,470 @@
+﻿using System;
+using System.IO;
+
+namespace Homework2
+{
+    class Program
+    {
+
+        //HW2
+        //
+        //Create a method for each operation:
+        //Sort an array
+        //Add element at the start of an array
+        //Add element at the end of an array
+        //Add element at any position of an array
+        //Remove element at the start of an array
+        //Remove element at the end of an array
+        //Remove element at a given index of an array
+        //Login: takes username and password.If they match a hidden username and password- a message "Logged in!" is printed.
+        //Username capitalization does not matter, password capitalization must match.
+
+        //HW 3
+        //
+        //You have a file called Users.txt. It contains usernames and passwords. A user open your application and they have 3 options:
+        //Login- enter their username and password, which should match any one line in Users.txt. Successful login will result in "Hello!" printed.
+        //Register- enter their username and password, which should append their credentials at the end of the file. In case of a duplicate user- try again.
+        //Exit- close the application
+        //Users.txt file should be at the same directory as the application start .exe.
+        //
+        //Errors
+        //If there is no Users.txt file- the application should not throw "UsersNotFoundException.
+        //If a Users.text file contains duplicate users, throw a DuplicateUserCredentialsException.
+        //
+        //Restrictions
+        //Don't use File class static methods.
+
+
+
+        //Variables
+        private static bool loggedIn = false; // Variable to track if user is "logged in"
+        private static string[][] userArray = new string[0][]; // array of user arrays with login info which is string[username,password]; Would rather use List<>
```

Comment:
If you are going to say `Would rather use List<>` you should explain why you cannot use `List<>`.  Here it's just a homework assignment and I'm guessing it's because you are using `userArray` for your HW2 work but in the future it might not be so easy to understand why.